lw/co/future_test.cpp

```diff
@@ -296,5 +359,75 @@ TEST(MakeFuture, RejectAValue) {
   EXPECT_THROW(f.await_resume(), InvalidArgument);
 }
 
+TEST(AwaitAll, AllResolve) {
+  Scheduler::this_thread().schedule([]() -> Task<void> {
+    std::tuple<int, float, double> res = co_await all(
+      make_resolved_future<int>(1),
+      make_resolved_future<float>(2.0f),
+      make_resolved_future<double>(3.14)
+    );
+    EXPECT_EQ(std::get<int>(res), 1);
+    EXPECT_EQ(std::get<float>(res), 2.0f);
+    EXPECT_EQ(std::get<double>(res), 3.14);
+  });
+  Scheduler::this_thread().run();
+  destroy_all_schedulers();
+}
+
+// TODO(alaina): Implement support for Future<void> in co::all.
```

Comment:
Is there an open issue for this? If so shouldn't you add which one it is here?

Reply:
There is now! #14